### PR TITLE
Juliana home page branch

### PR DESCRIPTION
**Pull request by julianaLeano**

### CODE REVIEW

**Review by MengtingCao — APPROVED** [review]
looks good

File: package.json [hunk 1]
```diff
@@ -2,12 +2,14 @@
   "name": "info-organizer",
   "private": true,
   "dependencies": {
-    "@popperjs/core": "2.11.6",
+    "@popperjs/core": "2.0.0-alpha.1",
```
Comment (jswood23) [inline package.json]: I don't think we should be changing this file in this PR.

**Review by jswood23 — CHANGES_REQUESTED** [review]
Looks good overall, just one change needed.

**Review by jswood23 — APPROVED** [review]
Great work!

**Pull request merged**